person.rb

```diff
@@ -1,14 +1,15 @@
 require './corrector'
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @age = age
     @name = name
     @parent_permission = parent_permission
     @correct_instance = Corrector.new
+    @rentals = []
   end
 
```

Comment:
A method for adding rentals should be present. Link to an example and inside the person itself should be sent as a parameter to create the new rental.